setup.py

```diff
@@ -122,6 +122,17 @@
     author_email='rbean@redhat.com',
     url='https://github.com/fedora-infra/fedmsg/',
     license='LGPLv2+',
+    classifiers=[  # https://pypi.python.org/pypi?%3Aaction=list_classifiers
+        'Development Status :: 3 - Alpha',
```

Comment:
I believe most of our apps have been following https://fedoraproject.org/wiki/Infrastructure/AppBestPractices#Version_numbering rather than entire semantic versioning.

So all you want to do would be doable, but would require a bump of the major version, from 0 to 1.